### PR TITLE
Add bundles for App Engine project conversion

### PR DESCRIPTION
**Pull request by chanseokoh**
For #348, add `.appengine.conversion` bundle and its test counterpart.

This is a mere skeleton, so no user-visible impact.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.conversion.test/src/com/google/cloud/tools/eclipse/appengine/conversion/PluginXmlTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.google.cloud.tools.eclipse.test.util.BasePluginXmlTest;
+import org.junit.Test;
+import org.w3c.dom.NodeList;
+
+public class PluginXmlTest extends BasePluginXmlTest {
+
+  @Test
+  public void testExtensionPoint() {
+    NodeList extensions = getDocument().getElementsByTagName("extension");
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.conversion.test/src/com/google/cloud/tools/eclipse/appengine/conversion/PluginXmlTest.java]: I seem to recall that Eclipse doesn't like plugins that define no extensions

Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.conversion.test/src/com/google/cloud/tools/eclipse/appengine/conversion/PluginXmlTest.java]: Maybe we should have a test to ensure there is no `plugin.xml`

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.conversion.test/src/com/google/cloud/tools/eclipse/appengine/conversion/PluginXmlTest.java]: I'll just remove `plugin.xml` from this PR. Will add back that when needed (soon).

**Comment by chanseokoh** [comment]
After discussing with @briandealwis ([related comment](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1188#issuecomment-271689159)), we agree on not adding a new bundle. Closing.

**Pull request closed without merging**